shell/platform/darwin/ios/platform_view_ios.mm

```diff
@@ -63,21 +63,24 @@
                                                        accessibility_bridge_.reset();
                                                        owner_controller_.reset();
                                                      }] retain]);
+    
+    // Do not call `NotifyCreated()` here - let FlutterViewController take care
+    // of that when its Viewport is sized.  If `NotifyCreated()` is called here,
+    // it can occasionally get invoked before the viewport is sized resulting in
+    // a framebuffer that will not be able to completely attach.
+}
 
+void PlatformViewIOS::attachView() {
   if (owner_controller_) {
     ios_surface_ =
-        [static_cast<FlutterView*>(owner_controller.get().view) createSurface:gl_context_];
+        [static_cast<FlutterView*>(owner_controller_.get().view) createSurface:gl_context_];
```

Comment:
All the changes to this file appear to be noops.  `owner_controller_` is set to `owner_controller` a few lines up.